MementoMori/MementoMori.Server/Controllers/CardDataController.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.AspNetCore.Mvc;
+namespace MementoMori.Server.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class CardDataController : ControllerBase
+    {
+        [HttpPost("postCard")]
+        public IActionResult PostInputData([FromBody] CardData data)
+        {
+            if (data == null || string.IsNullOrEmpty(data.Tags) || string.IsNullOrEmpty(data.Text))
```

Comment:
Since we are allowing only a selection of tags, we should also allow decks not have any tags if it does not math any.

@NaviRice56 @Vanielius1013 Thoughts?

Reply:
Wont fix for now